infrastructure/prod/ecosystem.config.js

```diff
@@ -2,18 +2,22 @@ module.exports = {
   apps: [
     {
       name: "app",
-      script: "npm",
-      args: "run prod:start",
+      script: "./node_modules/.bin/next",
+      args: "start -p 3001",
       cwd: "./packages/app",
+      instances: 2,
+      exec_mode: "cluster",
```

Comment:
This is the core of it. We run 2 instances in cluster mode, which will load balance requests between the two.